common/utils/src/main/resources/error/error-classes.json

```diff
@@ -5271,7 +5271,7 @@
   },
   "_LEGACY_ERROR_TEMP_1148" : {
     "message" : [
-      "Can only write data to relations with a single path."
+      "Can only write data to relations with a single path, paths are <paths>"
```

Comment:
```suggestion
      "Can only write data to relations with a single path but given paths are <paths>"
```

Reply:
Done. Thanks for your review. @HyukjinKwon